src/neptune/internal/utils/dependency_tracking.py

```diff
@@ -36,7 +36,7 @@ def log_dependencies(self, run: "Run") -> None:
 class InferDependenciesStrategy(DependencyTrackingStrategy):
     def log_dependencies(self, run: "Run") -> None:
         dependencies = []
-        dists = list(sorted(distributions(), key=lambda d: d.metadata["Name"]))
+        dists = list(distributions())
```

Comment:
Then we will not have an alphabetic order.
How about this? (List is actually not needed here)
```suggestion
        dists = sorted(distributions(), key=lambda d: d.metadata.get("Name", ""))
```

Reply:
I checked that the dependencies were in alphabetical order, even without sorting. Is this not the case at your end?

Reply:
I'm afraid not

```
cfgv==3.3.1
croniter==1.4.1
SQLAlchemy-Utils==0.38.3
pkgutil_resolve_name==1.3.10
matplotlib==3.7.1
starlette==0.27.0
pyrsistent==0.19.3
Markdown==3.4.3
xmltodict==0.13.0
chardet==3.0.4
regex==2023.3.23
pipreqs==0.4.13
requests-oauthlib==1.3.1
colorlog==6.7.0
accelerate==0.20.3
analytics-python==1.4.post1
packaging==23.0
s3transfer==0.7.0
webcolors==1.13
ptyprocess==0.7.0
nvidia-cublas-cu11==11.10.3.66
tqdm==4.65.0
deepdiff==6.6.0
gast==0.4.0
validators==0.18.2
sniffio==1.3.0
pexpect==4.8.0
sqlalchemy2-stubs==0.0.2a35
lightning==2.0.9.post0
executing==1.2.0
async-timeout==4.0.2
pycparser==2.21
```

Reply:
Updated my comment, as the previous proposition wouldn't work